admin/view-translations-post.php

```diff
@@ -34,7 +34,27 @@
 		}
 		?>
 		<tr>
-			<th class = "pll-language-column"><?php echo $language->flag ? $language->flag : esc_html( $language->slug ); // phpcs:ignore WordPress.Security.EscapeOutput ?></th>
+			<th class = "pll-language-column">
+			<?php
+			if ( $language->flag ) {
+					printf(
```

Comment:
I don't find that mixing HTML and PHP's output writing functions makes the clearest code to read. Could we just use one or the other?